src/themes/uswitch/theme.json

```diff
@@ -3874,14 +3874,14 @@
           "py": "xxs",
           "a": {
             "borderBottom": "none",
-            "mb": 0
-          },
-          "img": {
-            "width": 14,
-            "mt": 0,
-            "mr": "xxs"
+            "mb": 0,
+            "img": {
+              "width": 14,
+              "mt": 0,
+              "mr": "xxs"
+            }
           },
-          "h3": {
+          "h3[class*='Card']": {
```

Comment:
I don't think we get the card class in production, can you share a link to an example page?

Reply:
https://uswitch-leafeon-staging.external.rvu.cloud/dev/provider

If you inspect the text in sidebar that has a yellow background color you'll see that the class is there. I'm not sure how to increase specificity on that to be less hacky since if I remove that, the rich text block styling takes over.